programmers-coding-test/jaeseong/greedy/큰수의-법칙.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+const rl = require("readline-sync");
+let [ n, m, k ] = rl.question("n m k input please : \n>>").split(" ");
+let data = rl.question("array input please : \n>>").split(" ");
+console.log(`\n입력값 :${n},${m},${k}`);
+console.log(`배열값 :${data}`);
+
+function solution() {
+    data.sort();
```

Comment:
이렇게 정렬하면 문자의 아스키값으로 정렬이 됩니다.
파라미터로 숫자로 정렬되도록 지정해주어야합니다.
```suggestion
    data.sort((a, b) => b - a); // 내림차순으로 정렬하여 0번지 값을 빼오는 것이 가독성에 좋습니다.
```